### PR TITLE
WiRouterKeyRec Formula

### PR DESCRIPTION
**Pull request by giordanovi**
From Website http://www.salvatorefresta.net/tools/:
WiRouter KeyRec is a powerful and platform independent software to recover the default WPA passphrases of the supported router models (Telecom Italia Alice AGPF, Fastweb Pirelli, Fastweb Tesley, Eircom Netopia, Pirelli TeleTu/Tele 2).

### CODE REVIEW

File: Library/Formula/wirouter_keyrec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class WirouterKeyrec < Formula
+  homepage ""
```
Comment (mikemcquaid) [inline Library/Formula/wirouter_keyrec.rb]: Needs a homepage.

File: Library/Formula/wirouter_keyrec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  url "http://tools.salvatorefresta.net/WiRouter_KeyRec_1.1.2.zip"
+  version "1.1.2"
+  sha1 "3c17f2d0bf3d6201409862fd903ebfd60c1e8a2e"
+
```
Comment (mikemcquaid) [inline Library/Formula/wirouter_keyrec.rb]: Remove this blank line.

File: Library/Formula/wirouter_keyrec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  def install
+  	ENV.deparallelize
+    ENV.no_optimization
```
Comment (mikemcquaid) [inline Library/Formula/wirouter_keyrec.rb]: Why?

File: Library/Formula/wirouter_keyrec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+class WirouterKeyrec < Formula
+  homepage ""
+  url "http://tools.salvatorefresta.net/WiRouter_KeyRec_1.1.2.zip"
+  version "1.1.2"
+  sha1 "3c17f2d0bf3d6201409862fd903ebfd60c1e8a2e"
+
+
+  def install
+  	ENV.deparallelize
```
Comment (mikemcquaid) [inline Library/Formula/wirouter_keyrec.rb]: Can this be reported upstream and link to the upstream bug report? Thanks!

File: Library/Formula/wirouter_keyrec.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+	
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{prefix}",
+                          "--exec-prefix=#{prefix}"
+    system "make prefix=#{prefix}"
+    
+    (bin).install "build/wirouterkeyrec"
```
Comment (mikemcquaid) [inline Library/Formula/wirouter_keyrec.rb]: Is there no `make install` we can use?

Comment (giordanovi) [inline Library/Formula/wirouter_keyrec.rb]: No, won't install with make install

Comment (mikemcquaid) [inline Library/Formula/wirouter_keyrec.rb]: Is there a `make install`? If so, we should modify it so we can use it.

Comment (giordanovi) [inline Library/Formula/wirouter_keyrec.rb]: Formula modified to use make install.

**Comment by mikemcquaid** [comment]
Moving this into #36498.

**Pull request closed without merging**